src/applications/financial-status-report/tests/e2e/fsr-5655.cypress.spec.js

```diff
@@ -115,7 +115,9 @@ const testConfig = createTestConfig(
       'spouse/income/0': ({ afterHook }) => {
         afterHook(() => {
           cy.get(`input[name="root_spouseGrossSalary"]`).type('3500');
-          cy.get(`input[name="root_deductions_0_name"]`).type('Federal tax');
+          cy.get(`input[name="root_deductions_0_name"]`)
+            .type('Federal tax')
+            .blur();
```

Comment:
The `blur()` is to remove the dropdown so it doesn't cover the following input.

![image](https://user-images.githubusercontent.com/2008881/151041306-83135248-f544-4646-91b3-f3732333223a.png)

Otherwise we can get a failure like this:

![image](https://user-images.githubusercontent.com/2008881/151041994-3ec5e5c3-53d7-4bb9-b79d-be9dbabef1b6.png)

---

This seems like it might be a flaky test because even when it passes the "Amount" input is obscured:
![image](https://user-images.githubusercontent.com/2008881/151038716-41a708f7-854e-49fe-844f-8ec515a231b3.png)

Reply:
@craigwheeler @ScottyRJames can I get your :eyes: on this?

Reply:
Resolved by: https://github.com/department-of-veterans-affairs/vets-website/pull/20068